crates/forge/src/cheatcodes_hint_processor.rs

```diff
@@ -362,6 +365,46 @@ impl CairoHintProcessor<'_> {
                 Ok(())
             }
             "parse_json" => todo!(),
+            "spy_events" => {
+                let spy_on = match inputs.len() {
+                    0 => unreachable!(),
+                    1 => SpyOn::All,
+                    2 => SpyOn::One(contract_address_from_felt252(&inputs[1])?),
+                    _ => {
+                        let addresses_length = inputs[1].to_usize().unwrap();
+                        let addresses = Vec::from(&inputs[2..(2 + addresses_length)])
+                            .iter()
+                            .map(|felt| contract_address_from_felt252(felt).unwrap())
+                            .collect();
```

Comment:
I'm always unsure about these: Would it make sense to have `expect` with some messages instead of `unwrap`? Or is this overkill?

Reply:
I think this is overkill because those values come directly form the cairo code where they were casted from ContractAddress to felt252